wfc3tools/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - scipy
     - python x.x
 source:
-    git_tag: 1.3.1
+    git_tag: 1.3.3
```

Comment:
Hey @sosey, I see you tagged `v1.3.3` against the wfc3tools repo. If you do that you need to prefix the `git_tag` version with `v` here in the recipe.